src/Psy/TabCompletion/Matcher/ObjectMethodsMatcher.php

```diff
@@ -50,7 +50,10 @@ public function getMatches(array $tokens, array $info = array())
         return array_filter(
             get_class_methods($object),
             function ($var) use ($input) {
-                return AbstractMatcher::startsWith($input, $var);
+                return
```

Comment:
I know it's completely arbitrary, and the linter didn't even complain, but for a multiline return I prefer to wrap it in parens (as if it were a multi line array).

Reply:
Okay, will do that this afternoon.

Reply:
```php

                return
                    (AbstractMatcher::startsWith($input, $var)
                    // also check that we do not suggest invoking a super method(__construct, __wakeup, …)
                    && !AbstractMatcher::startsWith('__', $var));
```

Do you expect it like this?

Reply:
```php
return (
    AbstractMatcher::startsWith($input, $var) &&
    // also check that we do not suggest invoking a super method(__construct, __wakeup, …)
    !AbstractMatcher::startsWith('__', $var)
);
```